packages/ckeditor5-emoji/docs/_snippets/features/emoji-source.js

```diff
@@ -15,7 +17,10 @@ import { LinkImage } from '@ckeditor/ckeditor5-link';
 import ClassicEditor from '../build-classic.js';
 
 ClassicEditor.builtinPlugins.push(
+	Mention,
 	Emoji,
+	SpecialCharacters,
+	SpecialCharactersEssentials,
```

Comment:
I am still trying to determine if the Special characters feature is needed in the Emoji guide. I would remove it.